src/main/java/gyro/google/compute/ProjectMetadataItemResource.java

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright 2019, Perfect Sense, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gyro.google.compute;
+
+import com.google.api.client.googleapis.json.GoogleJsonResponseException;
+import com.google.api.services.compute.Compute;
+import com.google.api.services.compute.model.Metadata;
+import com.google.api.services.compute.model.Operation;
+import com.google.api.services.compute.model.Project;
+import gyro.core.GyroException;
+import gyro.core.GyroUI;
+import gyro.core.Type;
+import gyro.core.resource.Resource;
+import gyro.core.resource.Updatable;
+import gyro.core.scope.State;
+import gyro.core.validation.Required;
+import gyro.google.Copyable;
+
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Creates a project-wide metadata item. Set project-wide SSH keys by creating an item with the key ``ssh-keys``.
+ *
+ * Example
+ * -------
+ *
+ * .. code-block:: gyro
+ *
+ *     google::project-metadata-item project-metadata-item-example
+ *         key: "example-key"
+ *         value: "example-value"
+ *     end
+ */
+@Type("project-metadata-item")
+public class ProjectMetadataItemResource extends ComputeResource implements Copyable<Metadata.Items> {
+    private String key;
+    private String value;
+
+    /**
+     * The key of the metadata item. (Required)
```

Comment:
Even though we are not validating but we should mention the constrains for the `key` field